# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1523864088 -7200
#      Mon Apr 16 09:34:48 2018 +0200
# Branch stable
# Node ID fa9993b556176af287a0bc804fdb743b93e7033f
# Parent  316b75b0d886d982eef3557811231cc05dd60a5e
Update French translation for final time before 4.4 release (bug #53526)

diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -444,34 +444,34 @@ documentation viewer. Only help texts in
 the Console Widget will be available.</source>
         <translation>Impossible de préparer les données nécessaires
 pour le navigateur de documentation. Seule l&apos;aide
 dans la console (commande &quot;help&quot;) sera accessible.</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Find:</source>
-        <translation type="unfinished"></translation>
+        <translation>Rechercher :</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+1"/>
         <source>Search forward</source>
-        <translation type="unfinished"></translation>
+        <translation>Suivant</translation>
     </message>
     <message>
         <location line="+5"/>
         <location line="+1"/>
         <source>Search backward</source>
-        <translation type="unfinished"></translation>
+        <translation>Précédent</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Escape</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Contents</source>
         <translation>Contenu</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -3073,22 +3073,22 @@ voulez l&apos;affecter à la présente action et le retirer de la précédente ?</translation>
     <message>
         <location line="+4"/>
         <source>Run Menu</source>
         <translation>Menu Exécuter</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Import shortcuts from file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Importer des raccourcis depuis un ficher...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Export shortcuts into file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Exporter des raccourcis vers un fichier...</translation>
     </message>
     <message>
         <location line="-4"/>
         <location line="+5"/>
         <source>Octave Shortcut Files (*.osc);;All Files (*)</source>
         <translation>Fichier de Raccourcis Octave (*.osc);;Tous les fichiers (*)</translation>
     </message>
     <message>
@@ -3185,37 +3185,37 @@ Voulez sauvegarder les raccourcis actuel
     <message>
         <location line="+1"/>
         <source>Edit variables.</source>
         <translation>Éditer les Variables.</translation>
     </message>
     <message>
         <location line="+259"/>
         <source>Foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>Premier plan</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Background</source>
-        <translation type="unfinished">Arrière plan</translation>
+        <translation>Arrière plan</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Selected Foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>Premier plan sélectionné</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Selected Background</source>
-        <translation type="unfinished"></translation>
+        <translation>Arrière plan sélectionné</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternate Background</source>
-        <translation type="unfinished"></translation>
+        <translation>Arrière plan alterné</translation>
     </message>
     <message>
         <location line="+255"/>
         <source>Variable Editor Toolbar</source>
         <translation>Barre d&apos;outils de l&apos;éditeur de variables</translation>
     </message>
     <message>
         <location line="+4"/>
